community/kernel/src/main/java/org/neo4j/unsafe/impl/batchimport/executor/DynamicTaskExecutor.java

```diff
@@ -37,14 +37,15 @@
  */
 public class DynamicTaskExecutor<LOCAL> implements TaskExecutor<LOCAL>
 {
-    public static final ParkStrategy DEFAULT_PARK_STRATEGY = new ParkStrategy.Park( 10 );
+    public static final ParkStrategy DEFAULT_PARK_STRATEGY = new ParkStrategy.Park( 100, NANOSECONDS );
```

Comment:
Reducing the default park time from 10 milliseconds to 100 nanoseconds? No OS can context switch that quickly. You might as well be spinning or calling `Thread.yield()`. In Java 9 there will be an `Thread.onSpinWait()`.